data/database.py

```diff
@@ -951,6 +971,11 @@ class RepositorySearchScore(BaseModel):
     last_updated = DateTimeField(null=True)
 
 
+class RepositorySize(BaseModel):
+    repository = ForeignKeyField(Repository, unique=True)
+    size_bytes = BigIntegerField()
+
```

Comment:
Would it be useful to add a timestamp? Thinking if we need to ever need to recalculate this value, the last time it was updated should also be relevant. In most case, we shouldn't need to use a timestamp though, since we're keeping a running count on push.

Reply:
Even without a timestamp we can easily recalculate without much issue at this point I suggest we leave it off.

Reply:
Yes the separate table is to help avoid adding a column and creating issues with migration